src/stream/src/executor/batch_query.rs

```diff
@@ -52,6 +53,7 @@ where
                 PrefetchOptions::prefetch_for_large_range_scan(),
             )
             .await?;
+        let iter = iter.map_ok(|keyed_row| keyed_row.into_owned_row());
```

Comment:
Ideally we can let `collect_data_chunk` accept an iterator over any `impl Row`.

Reply:
In this PR we already change to support this. The reason why I introduced the `exact_size_iter` in `RowExt` to for `collect_data_chunk` to accept any `impl Row`. The `KeyedRow` here does not `Row` trait, and the `into_owned_row` is a method of itself and simply takes the internal `OwnedRow` out.

Reply:
You're right. 😄